include/megaapi_impl.h

```diff
@@ -393,21 +432,34 @@ class MegaBackupController : public MegaBackup, public MegaRequestListener, publ
     void setValid(bool value);
 };
 
-class MegaFolderDownloadController : public MegaTransferListener, public MegaRecursiveOperation
+class MegaFolderDownloadController : public MegaRequestListener, public MegaTransferListener, public MegaRecursiveOperation
```

Comment:
Do we need to inherit from MegaRequestListener?  We're not overriding any onRequest function so this is probably left over from earlier changes.